lib/core/src/lib/notifications/components/notification-history.component.ts

```diff
@@ -145,4 +156,8 @@ export class NotificationHistoryComponent implements OnDestroy, OnInit, AfterVie
             this.trigger.closeMenu();
         }
     }
+
+    badgeVisibility(): boolean {
```

Comment:
Could you rename that function as function should have verb as name?

Reply:
renamed to isBadgeVisible()